snapcraft/__init__.py

```diff
@@ -282,12 +282,14 @@
 from snapcraft._baseplugin import BasePlugin        # noqa
 from snapcraft._options import ProjectOptions       # noqa
 from snapcraft._help import topic_help              # noqa
+# TODO migrate to snapcraft.store
```

Comment:
This is the type of things that we read a month from now and we no longer remember why they made sense before and not now.
Can you please explain why, and add the date and your name?